pss/pss_test.go

```diff
@@ -70,6 +71,117 @@ func TestTrojanChunkRetrieval(t *testing.T) {
 
 }
 
+func TestPssMonitor(t *testing.T) {
+	var err error
+	ctx := context.TODO()
+
+	db, cleanupFunc := newTestDB(t, &Options{Tags: chunk.NewTags()})
+	defer cleanupFunc()
+
+	testTargets := [][]byte{
+		{57, 120},
+		{209, 156},
+		{156, 38},
+		{89, 19},
+		{22, 129}}
+	payload := []byte("RECOVERY CHUNK")
+	topic := trojan.NewTopic("RECOVERY")
+
+	var ch chunk.Chunk
+
+	pss := NewPss(db.localStore)
+
+	// call Send to store trojan chunk in localstore
+	if ch, err = pss.Send(ctx, testTargets, topic, payload); err != nil {
+		t.Fatal(err)
+	}
+
+	item, err := db.localStore.pullIndex.Get(shed.Item{
+		Address: ch.Address(),
+		BinID:   1,
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = db.localStore.Set(context.Background(), chunk.ModeSetSyncPull, ch.Address())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	checkTag(t, tag, 0, 1, 0, 1, 0, 1)
+
+}
+
+// CheckTag checks the first tag in the api struct to be in a certain state
+// TODO: reuse existing tag CheckTag instead of this
+func checkTag(t *testing.T, tag *chunk.Tag, split, stored, seen, sent, synced, total int64) {
```

Comment:
the body of this func seems to be an exact duplicate of the already exported `CheckTag` func from `chunk/testing/tag.go`

Reply:
removed